services/gateway/gRPC_client.py

```diff
@@ -0,0 +1,20 @@
+import grpc
+import user_pb2
+import user_pb2_grpc
+
+class GRPCClient(object):
+    def __init__(self):
+        self.host = '127.0.0.1'
```

Comment:
Make these (address and port) come from env at least

Reply:
Please check other services and make use of the _python-decouple_ library.